packages/storybook/.storybook/preview.js

```diff
@@ -3,14 +3,12 @@ import './style.scss';
 import '@department-of-veterans-affairs/formation/dist/formation';
 
 import '@department-of-veterans-affairs/component-library/dist/main.css';
-import {
-  applyPolyfills,
-  defineCustomElements,
-} from '@department-of-veterans-affairs/component-library';
 
-applyPolyfills().then(() => {
-  defineCustomElements();
-});
+import { VaAccordion } from '@department-of-veterans-affairs/component-library/dist/components/va-accordion';
+import { VaAccordionItem } from '@department-of-veterans-affairs/component-library/dist/components/va-accordion-item';
+
+customElements.define('va-accordion', VaAccordion);
+customElements.define('va-accordion-item', VaAccordionItem);
```

Comment:
This testing was to show that the new import & definition style would work.

On the accordion component's page:

![Screenshot of the storybook instance for this PR, showing that `<va-accordion>` web components appear and work as expected.](https://user-images.githubusercontent.com/2008881/201762027-5801ca10-0991-4827-a8c7-878e3bc682bc.png)

On the additional info page where we don't define `va-additional-info`:

![Screenshot of the storybook instance for this PR, showing that the browser doesn't do anything special for the `<va-additional-info>` web component tag since we've removed the custom elements definition for all of the web components](https://user-images.githubusercontent.com/2008881/201762112-90389376-d583-40a3-ac06-8c9359cbbe51.png)

---

In storybook, I couldn't get the following imports to work for some React/babel reason, but it worked as expected in `vets-website`:

```js

import {
  defineCustomElementVaAccordion
} from '@department-of-veterans-affairs/component-library/dist/components';

defineCustomElementVaAccordion();
```

![image](https://user-images.githubusercontent.com/2008881/201762642-aecd0903-ca20-49ed-be70-2d477c0164bf.png)

I tried updating storybook's babel config [based on this example](https://github.com/storybookjs/storybook/issues/17115#issuecomment-1033916240) but it only caused storybook to get stuck trying to build.